src/Subpages/Resources/TheProcessPage.jsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Container, Typography } from "@material-ui/core";
+import teamBuilding from "../../Assets/team_building.svg";
+import HeroPanel from "../../Components/HeroPanel";
+import { useResourcesStyles } from "../../Styles/useResourcesStyles";
+import ListItemMobileSnap from "../../ui-kit/ListItemMobileSnap";
+import SecondaryButton from "../../ui-kit/SecondaryButton";
+import { GatherDocuments } from "./GatherDocuments";
+import { LearnMore } from "./LearnMore";
+import { ScheduleHearing } from "./ScheduleHearing";
+
+const TheProcessPage = () => {
+  const classes = useResourcesStyles();
+  return (
+    <>
+      <Container>
+        <HeroPanel
+          title={"Resources"}
+          subtitle={
+            "The process of vacating a conviction is convoluted, making it difficult for most people to navigate. That’s why we have compiled a list of resources to help you get started."
+          }
+        />
+      </Container>
+      <Box className={classes.darkBlueBackground}>
+        <GatherDocuments />
+        <Container>
+          <Typography className={classes.headingStyle} variant="h2">
+            Step 2: Determine Eligibility
+          </Typography>
+          <ListItemMobileSnap
+            image={<Box height={175} component="img" src={teamBuilding} />}
+            textLeft={true}
+            text={
+              <Typography>
+                Once you have your records and forms gathered, use our
+                eligibilty calculator to determine if your conviction can be
+                vacated.
+              </Typography>
+            }
+          />
+          <SecondaryButton text={"Check Eligibility"} />
```

Comment:
Clicking this button doesn't do anything.

Reply:
Also this one doesn't need to open in a new tab ;)

Reply:
Thanks for catching that!